src/test/java/Homework18.java

```diff
@@ -0,0 +1,43 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+
+public class Homework18 extends BaseTest {
+    @Test(priority = 1, description = "Play a song and validate it is playing")
+    public void playSong() throws InterruptedException {
+
+        WebDriver driver = new ChromeDriver();
+        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
+
+        String url = "https://bbb.testpro.io/";
+        String urlHome = "https://bbb.testpro.io/#!/home";
+        driver.get(url);
+
+
+        WebElement loginField = driver.findElement(By.cssSelector("[type='email']"));
+        WebElement passwordField = driver.findElement(By.cssSelector("[placeholder='Password']"));
+        WebElement submitLogin = driver.findElement(By.cssSelector("[type='submit']"));
+
+
+
+        loginField.sendKeys("janezelenova@gmail.com");
+        passwordField.click();
+        passwordField.sendKeys("Floridaliving2023$");
+        submitLogin.click();
+
+        Thread.sleep(5000);
+
+
+       enterAllSongs();
+       selectSong();
+       enterButtonPlaySong();
+
+        Assert.assertTrue(isDisplayedPlayingSong());
+
+
```

Comment:
The code is not compiled! Please finish the homework!You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/221961357-80f9c7b6-9228-4d6d-ab7a-3346eb2fac8f.png)

![image](https://user-images.githubusercontent.com/104761482/221961493-81c2cacd-c0c6-47e5-a3fd-fab169c96daf.png)